wizeline/remix-project-lab
# feat: Refactor ProposalCard to take secondary actions out of main car…

# What does this PR do?


Refactor ProposalCard to take secondary actions out of main card area. I added the CardActions sections.

# Where should the reviewer start?



- app/core/components/EllipsisText/EllipsisText.styles.tsx
- app/core/components/EllipsisText/index.tsx
- app/core/components/ProposalCard/ProposalCard.styles.tsx
- app/core/components/ProposalCard/index.tsx
# How should this be manually tested?


1.- Be on the /projects
 You should see the cards and be able to click on the skills and the tiers links 

# Any background context you want to provide?
The ProposalCard component was changed to solve a browser error regarding rendering tags inside another tag. (changed the href in CardActionArea to an onClickEvent) which removed the ability to click + hold a key to open a new window/tab



# What are the relevant tickets?


https://github.com/wizeline/remix-project-lab/issues/232 [Refactor ProposalCard to take secondary actions out of main card area](https://github.com/wizeline/remix-project-lab/issues/232)
# Screenshots


Before
<img width="855" alt="Screenshot 2023-10-20 at 9 31 16" src="https://github.com/wizeline/remix-project-lab/assets/97254325/bf42b92e-6a25-4194-87e6-55a05b19d3be">

After
<img width="941" alt="Screenshot 2023-10-19 at 19 40 05" src="https://github.com/wizeline/remix-project-lab/assets/97254325/b4f2544a-587a-409a-9fab-c940ddb13c36">



# Questions



# Checklist
